src/service/user.service.js

```diff
@@ -0,0 +1,116 @@
+import { v4 as uuidv4 } from 'uuid';
+import { User } from '../modules/User.js';
+import { emailService } from './email.service.js';
+import { ApiError } from '../exeptions/api.error.js';
+import { ConsoleLoger } from '../untils/consoleLoger.js';
```

Comment:
There is a typo in 'ConsoleLoger'. It should be 'ConsoleLogger'. Ensure that the import statement and all usages are corrected.